mirroroperator/registrymirror.py

```diff
@@ -487,6 +501,26 @@ def generate_stateful_set(self):
                                 args=["-c", script],
                                 volume_mounts=generate_ca_certs_volume_mounts,
                                 resources=resources,
+                            ),
+                            client.V1Container(
+                                name="get-nameservers",
+                                image="busybox",
+                                command=["/bin/sh"],
+                                args=["-c", script_munge_nameservers],
+                                volume_mounts=[
+                                    client.V1VolumeMount(
+                                        name="nginx-config",
+                                        mount_path="/var/lib/conf.d",
+                                        read_only=False
+                                    ),
+                                    client.V1VolumeMount(
+                                        name="nameserversdir",
+                                        mount_path="/var/lib/nameserversdirtemp",
```

Comment:
not a big fan of this name `nameserversdirtemp`

Reply:
I was following the naming convention used internally but open to suggestions

Reply:
can you point to the naming convention you are following please

Reply:
Sure. I used the format shown in kube-proxy.yaml.jinja2 in kube-system

Reply:
Thanks, I'm not sure I'd consider that one example a convention though.

If we are keeping the container, i'd feel less queasy with `/tmp/nginx` as what is stored in that directory is nginx config and I feel that should be more obvious

Reply:
I will change the directory to /tmp/nginx

Reply:
I will make the volumeMount name change to nginx-config with mountpath /etc/nginx

Reply:
The volume name was changed to nginx-config-edited with mount path in the init container of /tmp/nginx